_specs/ecip-1088.md

```diff
@@ -0,0 +1,79 @@
+---
+lang: en
+ecip: 1088
+title: Phoenix EVM and Protocol Upgrades (Vanilla Option)
+status: Draft
+type: Meta
+author: /raw PONG _GHMoaCXLT (@q9f)
+created: 2020-02-24
+discussions-to: https://github.com/ethereumclassic/ECIPs/issues/262
+---
+
+### Simple Summary
+
+Enable the outstanding Ethereum Foundation _Istanbul_ network protocol upgrades on the Ethereum Classic network in a hard-fork code-named _Phoenix_ to enable maximum compatibility across these networks.
+
+This is the _vanilla_ option equaling to a vanilla EIP-1679 _Istanbul_ option of the Ethereum Foundation network.
+
+### Abstract
+
+Add support for a subset of protocol-impacting changes introduced in the Ethereum Foundation (ETH) network via the _Istanbul_ hardforks. The proposed changes for Ethereum Classic's _Phoenix_ upgrade include:
+
+- Add Blake2 compression function `F` precompile
+- Reduce alt_bn128 precompile gas costs
+- Add ChainID opcode
+- Repricing for trie-size-dependent opcodes
+- Calldata gas cost reduction
+- Rebalance net-metered SSTORE gas cost with consideration of SLOAD gas cost change
+
+This document proposes the following blocks at which to implement these changes in the Classic networks:
+
+- `999_983` on Mordor Classic PoW-testnet (April, 2020)
```

Comment:
I think this should be `< 778507`, which was the original Aztlan ECIP1061 activation number.

Reply:
what's the reasoning? having a higher block number gives is much more time preparing clients and releases. having a block number that lives in the past forces us to push through consensus and client releases.

Reply:
`778_507` is the latest definitely-viable number. The number may be able to be higher, depending on what was run on the network. If no one used or interacted with opcodes or pricing modified by the Aztlan+/PhoenixFix configurations, the block number may be able to be later.